cacheobject/object.go

```diff
@@ -109,10 +93,29 @@ func CachableObject(obj *Object, rv *ObjectResults) {
 	if obj.ReqDirectives != nil && obj.ReqDirectives.NoStore {
 		rv.OutReasons = append(rv.OutReasons, ReasonRequestNoStore)
```

Comment:
shouldn't this also move to CachableResponseObject?

Reply:
I hesitated but I don't think so, it's related to the request, not to the response.